test/TEST-04-JOURNAL/test.sh

```diff
@@ -7,4 +7,11 @@ TEST_DESCRIPTION="Journal-related tests"
 # shellcheck source=test/test-functions
 . "${TEST_BASE_DIR:?}/test-functions"
 
+test_append_files() {
+    local workspace="${1:?}"
+
+    mkdir -p "$workspace/test-journals/"
+    cp -av "${TEST_BASE_DIR:?}/test-journals/"* "$workspace/test-journals/"
```

Comment:
any chance you can make these test files (way) smaller, or better generate them on the fly? Currently it's an extra 56MB added to the git repo, which is quite a bit

Reply:
i'd really like this to be real-life examples.

anyway, will compress them, that makes this 1M

Reply:
Can you do zstd instead of gz? Total goes from 2.1M to 1.8M